### PR TITLE
[PERF-238] Run Pipeclean Tests in Latest Locust

### PR DESCRIPTION
**Pull request by vimayya**

### CODE REVIEW

File: src/edfi-performance-test/edfi_performance_test/performance_tester.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        start = time.time()
+
+        set_config_values(args)
+
+        # setup Environment and Runner
+        env = Environment(user_classes=[DummyUser], host=args.baseUrl)
+        env.create_local_runner()
+
+        # start a WebUI instance
+        env.create_web_ui("127.0.0.1", 8089)
```
[the remark this answers is not in the document]
Comment (stephenfuqua) [reply src/edfi-performance-test/edfi_performance_test/performance_tester.py:41]: Separate ticket: https://tracker.ed-fi.org/browse/PERF-268

File: src/edfi-performance-test/edfi_performance_test/performance_tester.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+        # wait for the greenlets
+        env.runner.greenlet.join()
+
+        # stop the web server for good measures
+        if env.web_ui:
+            env.web_ui.stop()
+
+        # for running tests in a debugger
+        # run_single_user(DummyUser)
```
[the remark this answers is not in the document]
Comment (stephenfuqua) [reply src/edfi-performance-test/edfi_performance_test/performance_tester.py:73]: Separate ticket: https://tracker.ed-fi.org/browse/PERF-268

File: src/edfi-performance-test/edfi_performance_test/performance_tester.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+from locust.stats import stats_printer, stats_history
+
+from edfi_performance_test.helpers.main_arguments import MainArguments
+from edfi_performance_test.helpers.config import set_config_values
+from edfi_performance_test.tasks.pipeclean.pipeclean_tests import DummyUser
+
+logger = logging.getLogger(__name__)
+
+
+async def run(args: MainArguments) -> None:
```
[the remark this answers is not in the document]
Comment (stephenfuqua) [reply src/edfi-performance-test/edfi_performance_test/performance_tester.py:28]: I created a separate ticket for this: https://tracker.ed-fi.org/browse/PERF-269

**Review by stephenfuqua — APPROVED** [review]

File: src/edfi-performance-test/edfi_performance_test/api/basic_client/ed_fi_basic_api_client.py [hunk 4]
```diff
@@ -9,25 +9,26 @@
 import json
```
[the remark this answers is not in the document]
Comment (vimayya) [reply src/edfi-performance-test/edfi_performance_test/api/basic_client/ed_fi_basic_api_client.py:9]: used by change query tests in the old suite. we can remove later if we don't end up using.

**Pull request merged**